multinode_integration_tests/tests/verify_bill_payment.rs

```diff
@@ -233,7 +233,7 @@ fn verify_bill_payment() {
     assert_balances(
         &contract_owner_wallet,
         &blockchain_interface,
-        "99995074522000000000",
+        "99995231980000000000",
```

Comment:
The reason for this is because the multinode tests were not run before creating the second review.
It had mistakenly believed `./ci/all` ran multinode tests also. 😅

This is because of the changes we made to Gas fees (gwei to wei) from the first review.

Reply:
Ah, if that's how it is then it is probably all right.